src/Pages/Playground/components/playgrounApiDocs.tsx

```diff
@@ -0,0 +1,587 @@
+import {
+  Component,
+  createEffect,
+  createResource,
+  Switch,
+  Match,
+} from "solid-js";
+import { For, Show, createSignal } from "solid-js";
+
+
+// const schemaSend = async (item: string) =>  {
+//   return await import('../../../schema/'+ item+ '/send.json')
+// };
```

Comment:
commented code, remove or document TODO pls

Reply:
sure I will add TODO for it

Reply:
thanks